lib/functoria/key.mli

```diff
@@ -182,7 +182,7 @@ val v : 'a key -> t
 (** [v k] is the [k] with its type hidden. *)
 
 val abstract : 'a key -> t
-  [@@ocaml.deprecated "Use Functoria.Key.v."]
```

Comment:
@Julow Is this expected? The annotation is supposed to be attached to the top-level item, right?

Reply:
The attribute is unindented but still attached to the same item, yes.

Reply:
what's the rationale for changing this?

Reply:
This is changed here: https://github.com/ocaml-ppx/ocamlformat/pull/2317

It was initially implemented for the Janestreet profile and I thought less indentation is generally better so I changed the default profile too. Do you prefer the previous formatting ?

Reply:
Ha ok, that matches ocp-indent behavior then? If yes, I'm fine with the changes, that's a good enough rational.

Reply:
It does. The main reason was that less indentation is generally better and this is useless indentation.

Reply:
Was it reverted in the new version you just pushed?

Reply:
No, the attributes have been removed in cf46b243318265024c88ceb86dc47ca16c3087cc and I rebased this PR on top.